scraper.py

```diff
@@ -0,0 +1,169 @@
+import selenium
```

Comment:
Can you rename this file to simply `atlanta.py` and move it to: https://github.com/CouncilDataProject/cdp-scrapers/tree/main/cdp_scrapers/instances

the `instances` module.

Reply:
Bump this comment again. I will show you all why next Tuesday.